sections_text/README.md

```diff
@@ -4487,3 +4487,28 @@ Here we use some of the classes of the [card](https://materializecss.com/cards.h
 - Finally, test on your browser
 
 Finally, we finished our goals for this app but this is just the beginning of our journey because will be your job to expand and refactor the app. Good look and thanks for reading this.
+
+## Git workflow
+
+On this repository I going to implement a `git workflow` that will allow us to accomplish our goals on a consitent and productive manner using git in the most effective way that we can. I defined the following specifications
+
+- Our main branch will be call `master`
```

Comment:
Adding a graphical representation of the flow could help others quickly and clarify any doubts

Reply:
Let me check how I can generate an image of the workflow. Nice catch!!

Reply:
Done!!!